docs/fundamentals/code-analysis/style-rules/ide0073.md

```diff
@@ -40,12 +40,15 @@ This style rule concerns providing a file header at top of source code files. Th
 | **Option values** | non-empty string - Prefer the string as required file header.<br /><br /> `unset` or empty string - Do not require file header. |
 | **Default option value** | `unset` |
 
+You can include `{fileName}` as a placeholder for the file name.
```

Comment:
Should we put this alongside the non-empty string description for the option value?